Sources/Run/main.swift

```diff
@@ -1,3 +1,9 @@
 import App
+import Vapor
 
-try app(.detect()).run()
+var env = try Environment.detect()
+try LoggingSystem.bootstrap(from: &env)
+let app = Application(env)
+defer { app.shutdown() }
+try configure(app)
+try app.run()
```

Comment:
I don't know what those lines do, but I'm guessing they are comming from the documentation ;)

Reply:
It looks they are from deleted file `Sources/App/app.swift `

Reply:
yep, this is the new way of setting up the app :) It's basically from the template